source/direct_debit/index.html.md.erb

```diff
@@ -5,285 +5,259 @@ weight: 80
-a new payment. This requires users to re-enter their account details.
+Refer to the <a
+href="https://govukpay-api-browser.cloudapps.digital/#newpayment"
+target="blank">GOV.UK Pay API browser</a> or the <a href="https://github.com/alphagov/pay-publicapi/blob/master/swagger/swagger.json" target="blank">Swagger file</a> for more
+information on the API response.
 
-## Variable payments using Direct Debit
+### 3. The user provides their bank account details
 
-You can collect payments of variable amounts at any time by using a variable agreement.
-For example you may want to collect:
+When your service redirects the user to `next_url`, the user will see a **Set up a Direct Debit** page. The user enters their bank account details and selects **Continue**.
 
-* money each time a service is provided
+GOV.UK Pay checks with GoCardless that the user’s account number and sort code match a real bank account.
 
-* payments where the amounts vary according to service usage
+### 4. GOV.UK Pay presents the user with a confirmation screen
 
-* regular payments where you do not know in advance how much each payment will be
+The screen presents the user’s account details. The user grants their consent
+by selecting __Confirm__.  GOV.UK Pay sends an email to the user to confirm
+that their mandate has been set up. If the mandate then fails, the
+GOV.UK Pay team will send an email to update the user that their mandate could
+not be set up.
 
-You could also use this functionality to simulate a recurring or
-scheduled Direct Debit payment, by sending an API call on a set schedule.
-Users would be emailed each time a payment is collected against their mandate.
+### 5. GOV.UK Pay securely passes the user's details on to GoCardless
 
-### Variable Direct Debit payment process
+GoCardless submits a request to BACS to set up a mandate, and
+sends a mandate ID to GOV.UK Pay. At this point, you can send a payment
+request.
 
-These steps happen in order.
+### 6. GOV.UK Pay returns the user to your service using the `return_url`
 
-#### 1. GOV.UK Pay creates the agreement
+### 7. Call the API to check the status of the mandate
 
-API call:  `POST /v1/agreements/`
+API call: `GET /v1/directdebit/mandates/<mandateid>`
 
-Your service must specify:
+The API response includes a `state` object. This includes the following parameters:
 
-* a `return_url` where GOV.UK Pay will direct the user when they have finished setting up the agreement
+- [`status`](#mandate-statuses)
+- [`details`](https://developer.gocardless.com/api-reference/#events-mandate-causes), which explains the reason for the `status`
 
-* the `agreement_type` (`on demand`)
+You should use these `status` and `details` parameters to display an appropriate final page to your user.
 
-* a `reference` (optional) - your reference for this agreement
+The `status` will be `pending` until the user's bank confirms the mandate. The mandate confirmation process takes 2 working days.
```

Comment:
This might want to be qualified (it will only be `pending` if everything has gone well).

Reply:
I might have misunderstood the flow but - at this point in the chronology it seems the status could only be `pending`? Because the user has successfully filled in their details on GOV.UK Pay?

Reply:
The user might have cancelled or something. In theory I think it could also fail as soon as we send the details to GoCardless too.

At the point where the service is calling this API, all they know is the user has been returned from Pay. They won’t yet know if the user journey was successful.